style.css

```diff
@@ -343,9 +343,159 @@ section#grave-care {
 
 .grave-care-overlay {
   background-color: rgba(0, 0, 0, 0.72);
-  position: absolute;
   width: 100%;
   height: 100%;
   padding: 5rem 15rem;
   text-align: center;
 }
+
+/*---Section Documents---*/
+
+section#speaker {
+  background-color: #fff;
+  display: flex;
+  padding: 60px 0px;
+}
+
+/*---Section Contact---*/
+
+section#contact {
+  padding: 60px 0px;
+}
+
+h3.contact-header {
+  text-align: left;
+  color: #fff;
+  font-size: 3.5rem;
+  margin-bottom:4rem;
+  font-family: var(--header-font);
+}
+
+#contact-container {
+  max-width: 1140px;
```

Comment:
...and here too, rems instead of px? :)

Reply:
you love rems more than me ;)

Reply:
Posssibly! :D Anyways, it's not mandatory here if you have a reason not to use them. I'd just encourage mindful usage of both units. :) Rems are better for accessibility reasons, px however play better when it comes to fine details. :)

Here are a few videos of Kevin Powell (great channel btw). You may like to have a look and decide for yourself what and when plays best for a given situation:
 - [Are you  using the right CSS units?](https://www.youtube.com/watch?v=N5wpD9Ov_To&t=12s&ab_channel=KevinPowell)
- [CSS em and rem explained](https://www.youtube.com/watch?v=_-aDOAMmDHI&ab_channel=KevinPowell)
- [Why you shouldn't set font-sizes using em](https://www.youtube.com/watch?v=pautqDqa54I&ab_channel=KevinPowell)

Reply:
Thank you ❤️